frontend/src/types/dashboard.ts

```diff
@@ -0,0 +1,25 @@
+export interface Question {
+  type: 'SHORT_ANSWER' | 'LONG_ANSWER' | 'MULTIPLE_CHOICE' | 'DROPDOWN';
+  question: string;
+  choices: {
+    choice: string;
+    order_index: number;
```

Comment:
해당 API 스펙 문서에서 `key`값이 모두 `camelCase`로 변경되었어요. 이 부분도 반영해 주시면 감사하겠습니다!

```suggestion
    orderIndex: number;
```